resources/frameworks.yaml

```diff
@@ -46,6 +46,16 @@ TunedRandomForest:
 #      n_estimators: 500
 
 ### AutoML frameworks
+AutoGluon:
+  version: "0.0.8"
```

Comment:
Just released autogluon==0.0.9 as an emergency fix to sklearn==0.23 breaking skopt, which resulted in autogluon failing to import. Would recommend 0.0.9 to avoid having to specify sklearn version